README.md

```diff
@@ -93,7 +101,7 @@ docker tag okbr/notebooks:0.1 notebooks
 docker-compose up -d
 ```
 
-*NOTE: Expected to have docker-compose file in the directory where we run above command*
+<NOTE: Expected to have docker-compose file in the directory where we run above command>
```

Comment:
It's just an > in the beginning of every line.
https://guides.github.com/features/mastering-markdown/#syntax

Reply:
Updated again. Please check now